portal/add_edit_event_user.php

```diff
@@ -64,6 +64,23 @@
     exit();
 }
 
+if (!empty($_POST['form_pid'])) {
+    if ($_POST['form_pid'] != $_SESSION['pid']) {
+        echo js_escape("error");
```

Comment:
also should have `exit;` here
in addition to doing above (should definitely exit with error since means the patient is up to no good) also would change all `$_POST['form_pid']` to `$_SESSION['pid']` in this script